concrete-core/src/backends/core/implementation/engines/destruction.rs

```diff
@@ -0,0 +1,323 @@
+use crate::backends::core::implementation::engines::CoreEngine;
+use crate::backends::core::implementation::entities::{
+    Cleartext32, Cleartext64, CleartextVector32, CleartextVector64, FourierLweBootstrapKey32,
+    FourierLweBootstrapKey64, GlweCiphertext32, GlweCiphertext64, GlweCiphertextVector32,
+    GlweCiphertextVector64, GlweSecretKey32, GlweSecretKey64, LweBootstrapKey32, LweBootstrapKey64,
+    LweCiphertext32, LweCiphertext64, LweCiphertextVector32, LweCiphertextVector64,
+    LweKeyswitchKey32, LweKeyswitchKey64, LweSecretKey32, LweSecretKey64, Plaintext32, Plaintext64,
+    PlaintextVector32, PlaintextVector64,
+};
+use crate::backends::core::private::math::tensor::AsMutTensor;
+use crate::specification::engines::{DestructionEngine, DestructionError};
+
+impl DestructionEngine<Cleartext32> for CoreEngine {
+    fn destroy(&mut self, entity: Cleartext32) -> Result<(), DestructionError<Self::EngineError>> {
+        unsafe { self.destroy_unchecked(entity) };
+        Ok(())
+    }
+
+    unsafe fn destroy_unchecked(&mut self, _entity: Cleartext32) {}
```

Comment:
Just a stupid question: why is it `_entity` here, couldn't we use `entity` simply?

Reply:
without the `_`, rustc would display a warning about the variable `entity` not being used